usb_autorunner_profiles/deploy/run.sh

```diff
@@ -40,9 +40,14 @@ echo "⚙️  Upload the distro..."
 # Sending distro to volume
 $PWD/utils/upload-files.sh $REGISTRY_IP/mdlh/alpine-rsync:3.11-3.1-1 $PWD/distro/ distro-pvc
 
-echo "🧽 Delete the 'openmrs' pod (will be recreated right after)"
-POD_NAME=$($kubectl_bin get pod -l app=openmrs -o jsonpath="{.items[0].metadata.name}" -n $NAMESPACE)
-$kubectl_bin delete pod $POD_NAME -n $NAMESPACE
+echo "🧽 Delete the current 'openmrs' pod"
+$kubectl_bin delete pods -l app=openmrs -n $NAMESPACE
```

Comment:
@chinzou , don't we need to add `if-exists`or smtg? So that the command does not fail if the pod doesn't not exist.

Reply:
When deleting pods with label it will not fail if not found